src/toxcore/friend_connection/packet/share_relays.rs

```diff
@@ -0,0 +1,82 @@
+/*! ShareRalays struct
+*/
+
+use toxcore::binary_io::*;
+use toxcore::state_format::old::*;
+
+/** ShareRelays is a struct that holds 3 TCP relays as a PackedNode format.
+
+This packet is used to share relay info. between two friends.
+
+Serialized form:
+
+Length    | Content
+--------- | ------
+`1`       | `0x11`
+variable  | 3 PackedNodes
```

Comment:
Length is not variable in this case.

Reply:
Size of PackedNode is variable, so size of this struct is variable.

Reply:
ok, got it.